src/js/components/Marathon.jsx

```diff
@@ -371,7 +421,7 @@ var Marathon = React.createClass({
   getAppPage: function () {
     var activeApp = this.state.collection.get(this.state.activeAppId);
     if (!activeApp) {
-      return;
+      return null;
```

Comment:
I disagree. If I understood correctly, `getAppApage` is supposed to return something "renderable". In React that is supposed to be either a component, or `null`/`false`. 
From the docs:
> You can also return null or false to indicate that you don't want anything rendered. Behind the scenes, React renders a <noscript> tag to work with our current diffing algorithm. When returning null or false, React.findDOMNode(this) will return null.